docs/design/chart_format.md

```diff
@@ -347,7 +323,7 @@ The format for a repository has packages stored in a flat namespace, with versio
 * Brevity is desirable when developers have occasion to type these URLs directly, as is done in chart/template references (see, for example, the [S3 API](http://docs.aws.amazon.com/AmazonS3/latest/dev/UsingObjects.html)).
 
 ### Aside: Why Not Git(Hub) or VCS?
-In their first incarnations, both Helm and DM used GitHub-backed repositories for released charts. The Helm team believes we made a huge mistake by going this route. This method suffers from some severe limitations (listed below). The proposal above is designed to work around those limitations.
+In their first incarnations, both Helm and DM used GitHub-backed repositories for released charts. The Helm team believes we made a huge mistake by going this route. This method suffers from some severe limitations (listed below). Our repository design does not have these limitations.
```

Comment:
Second sentence... the Helm team. Reads like us vs. them. We were two teams, both of whom used Github. We are now one team and this design represents the thinking of the unified team. Please update the text accordingly.